### ISSUE TITLE
History++: Crash while open TabSRMM

### ISSUE THREAD

**Issue by facled** [miranda-ng/miranda-ng#1993]
If i open chat window than Miranda NG crashes every time.
But if i disable History++ than it works but cant use History++ Plugin.

1 or 2 days ago it works normally.

[VersionInfo.txt](https://github.com/miranda-ng/miranda-ng/files/3310901/VersionInfo.txt)
crash: [historypp.txt](https://github.com/miranda-ng/miranda-ng/files/3310894/historypp.txt)

**Comment by georgehazan** [comment]
@facled unfortunately h++ is written in Delphi, so it has no symbols in the stack...
is that possible that you disable h++, open tabSRMM's window with this contact and send me a screenshot of it? if it's private, send it via email to ghazan at miranda point im

**Comment by georgehazan** [comment]
@facled btw... very interesting experiment is to downgrade Miranda to 32 bits using a checkbox in Plugin Updater and test this contact again

**Comment by facled** [comment]
i test with h++ activated a little bit, contacts with 0 message can i open chat window without crashes.
but contacts with more messages miranda crash.
there is nothing special written, normal text and unicode emoji.
because downgrade to 32 bit, i got same crash.

at the moment i deactivate h++
i cant read russian board part but it confuse me that im the only have this problem.

**Comment by georgehazan** [comment]
yes, I use h++ too and never saw such a bug. it seems that h++ wrongly decodes one of these messages in this particular history. can you export this history somehow and send to me via email? I guarantee full privacy of these messages

**Comment by facled** [comment]
sorry, i dont know how i can export all messages in original decode standard from all contact between now and 7 days ago.

**Comment by georgehazan** [comment]
@facled for example, you can try to use BasicHistory plugin
at least, even if it crashed, I can debug it :-)

**Comment by facled** [comment]
thank you, with basichistory miranda not crash.

**Comment by georgehazan** [comment]
@facled no, I meant that you could export history using BasicHistory plugin :)
